masterlist.yaml

```diff
@@ -3768,6 +3768,11 @@ plugins:
           - '[Horizon](https://www.nexusmods.com/fallout4/mods/17374/)'
           - '[Sim Settlements 2 - Horizon Patch](https://www.nexusmods.com/fallout4/mods/54186/)'
         condition: 'active("Z_Horizon.esp") and not active("SS_Addon_Horizon.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[D.E.C.A.Y](https://www.nexusmods.com/fallout4/mods/2500/)'
+          - '[D.E.C.A.Y - Sim Settlements 2 Patch Instructions](https://www.nexusmods.com/fallout4/mods/2500?tab=posts)'
+        condition: 'active("D.E.C.A.Y.esp")'
```

Comment:
Considering the contents of `patch3rdParty`, it doesn't fit here as it implies a patch that users can download:

```suggestion
```

There's not really a localized message that could convey this information, but you could add a conditional incompatibility & link users to the instructions via the `detail` key. There's also the issue that just linking to the posts tab will cause issues if or when the post (assuming it's [this one](https://forums.nexusmods.com/index.php?/topic/3445720-decay/page-88#entry102651758)) is no longer at the top of the first page (or on the first page at all). You could accomplish this by adding the following incompatibility (I can't do this in a suggestion):

```yaml
      - name: 'D.E.C.A.Y.esp'
        detail: 'See [this guide](https://forums.nexusmods.com/index.php?/topic/3445720-decay/page-88#entry102651758) to make them compatible.'
        condition: 'checksum("D.E.C.A.Y.esp", 369D7560) or checksum("D.E.C.A.Y.esp", 1BA826DE) or checksum("D.E.C.A.Y.esp", 98A7E896)'
```

The three checksums are for the three versions of the plugin available at the plugin's page, unedited. Unfortunately, there's no way to tell for sure that the user has followed the instructions; we can only tell when they haven't edited the file at all & therefore haven't made the compatibility edit. False negatives are preferable over false positives as the latter tend to cause more confusion & issues for users.

Reply:
See updated commit below.